packages/eslint-config-custom/index.js

```diff
@@ -50,5 +53,11 @@ module.exports = {
   ignorePatterns: ['dist/*'],
   settings: {
     react: { version: 'detect' },
+    tailwindcss: {
+      // Ensures that ESLint knows about our Tailwind config, such as color
+      // names, etc. Without this, ESLint integrations (such as with the VSCode
+      // extension) break.
+      config: require.resolve('@penumbra-zone/tailwind-config'),
```

Comment:
And this is one of the other main changes: ensuring ESLint knows about our Tailwind config, which makes for better integration with VSCode.